src/redteam/getHarmfulTests.ts

```diff
@@ -43,8 +45,123 @@ export const HARM_CATEGORIES: { [key: string]: string } = {
   //'locale_specific_illegal (e.g. hate speech in Germany, alcohol in Saudi Arabia)',
 };
 
+interface HarmfulCategory {
+  key: string;
+  description: string;
+  examples: string;
+}
+
+// We use our default provider for some categories that llama guard does a particularly poor job at. We define
+// these categories here and provide sample prompts for them.
+const REDTEAM_MODEL_CATEGORIES: HarmfulCategory[] = [
+  {
+    key: 'harmful:intellectual-property',
```

Comment:
I think we need something better for IP violations specifically - I'm not sure that writing a derivative story without attribution is a violation.  The concern is direct plagiarism/ability to reproduce copyrighted works verbatim.

Also, I notice that most of the generations include something like "write X with unauthorized use of characters".  The prompt shouldn't admit upfront that it's doing a "bad" thing.